content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+<a id="introduction"></a>
+## Introduction
+
+> TEXT TO WRITE:
+> 
+> * talk about Kubeflow being a "Kubernetes" platform, with multiple components (not an "app")
+>    * link to kubeflow-overview for more details
+>    * (TODO) add a link to "Reference"/"Kubeflow Versioning Policies" in overview docs
+> * short discussion about how Kubeflow is versioned, 
+>    * components in Kubeflow can maintain their own versions, independent of the project-wide versions (like 1.2, 1.3)
+> 
+> * short discussion about the two "paths" for installing kubeflow "XXXXX" and "YYYYY"
+
+<a id="packaged-distributions"></a>
+## Packaged distributions
+
+> TEXT TO WRITE:
+> 
+> * distributions are separate from the kubeflow community
+>    * none are currently endorsed or certified
+>    * please reach out to the maintainers to get support
+> * the method for installing can vary from distribution to distribution, please consult the docs for your chosen distribution    
+
```

Comment:
Proposal (edited from #2625):

> Kubeflow distributions are developed and supported by their respective maintainers. The community does not currently endorse or certify any of the distributions, fostering openness and healthy competition. See the table below for a list of options and links to documentation.

@thesuperzapper wdyt? maybe too much? :)

Reply:
I have included something very similar to your suggestion